internal/backuprestore/restore.go

```diff
@@ -122,9 +122,14 @@ func (h *BRHandler) StartOrTrackRestore(ctx context.Context, restores []*velerov
 	return rt, nil
 }
 
-func (h *BRHandler) LoadRestoresFromOadpRestorePath() ([][]*velerov1.Restore, error) {
+func (h *BRHandler) LoadRestoresFromOadpRestorePath(ibu *lcav1alpha1.ImageBasedUpgrade) ([][]*velerov1.Restore, error) {
 	var sortedRestores [][]*velerov1.Restore
 
+	if len(ibu.Spec.OADPContent) == 0 {
```

Comment:
I don't think it's really needed since the restore dir won't be created if backup/restore is not needed.

Reply:
reverted